src/Compilers/VisualBasic/Test/CommandLine/CommandLineTests.vb

```diff
@@ -2104,6 +2104,7 @@ End Module").Path
             Assert.Equal(LanguageVersion.VisualBasic16, LanguageVersion.VisualBasic16.MapSpecifiedToEffectiveVersion())
             Assert.Equal(LanguageVersion.VisualBasic16_9, LanguageVersion.VisualBasic16_9.MapSpecifiedToEffectiveVersion())
             Assert.Equal(LanguageVersion.VisualBasic17_13, LanguageVersion.VisualBasic17_13.MapSpecifiedToEffectiveVersion())
+            Assert.Equal(LanguageVersion.VisualBasic17_13, LanguageVersion.Default.MapSpecifiedToEffectiveVersion())
```

Comment:
Looking at the comment below points to some instructions for adding a new VB language version. 
But the first bullet is not applicable (no UpgradeProject for VB) and the last bullet is no longer applicable (that document doesn't include specific version numbers since we added `/LangVersion:?`). Would you mind removing them?
 
```
  ' When a new version is added, this test will break. This list must be checked:
  ' - update the "UpgradeProject" codefixer (not yet supported in VB)
  ' - update all the tests that call this canary
  ' - update the command-line documentation (CommandLine.md)
```